src/navigation/navigation.tsx

```diff
@@ -184,6 +187,14 @@ const MainStackNavigator = (): JSX.Element => {
           </>
         )}
       />
+      <Stack.Screen
+        name={MainRoutesEnum.SHARE}
+        children={() => (
+          <>
```

Comment:
we do not need the empty tags, because there is only 1 child

Reply:
I have removed these tags